LINK/bin/miqnet.sh

```diff
@@ -279,13 +281,12 @@ GATEWAY=${3}
 NM_CONTROLLED=no
 DNS1=${4}" >> ${cfg}.tmp
   mv ${cfg}.tmp $cfg
+  restorecon $cfg
   log "set_redhat_static: cfg new: $(cat ${cfg})"
 
   # Truncate the temporary io file
   :> $TMP_IO
 
-  /sbin/restorecon -R /etc/sysconfig
```

Comment:
@carbonin I assume this is doing recursive restorecon on /etc/sysconfig when we only needed /etc/sysconfig/network-scripts/ifcfg-eth0, which is done above? Are there other files in /etc/sysconfig that we might have to be doing restorecon on that we still need to do it on?